README.md

```diff
@@ -47,7 +47,7 @@ This initial work supports Kaniko running on Minikube. Has potential to work on
 ## The Next Steps
 
 Despite this code being a PoC, it can evolve to do more and abstract the build stage for a shareable use among both projects.
```

Comment:
both projects, Camel-K and Kogito?
if so, please mention it.

Reply:
ok